scripts/crowbar-backup-cronjob

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# example backup script to be added in /etc/cron.daily/
+dir=/tmp/backup
```

Comment:
Why do we need to cd into that directory? If you need a scrstch space, better use one that is only root writeable